src/db_cache/mod.rs

```diff
@@ -238,47 +242,94 @@ impl CachedEntry {
 pub struct DbCacheWrapper {
     stats: DbCacheStats,
     cache: Arc<dyn DbCache>,
+    last_err_log: Mutex<Option<Instant>>
```

Comment:
Nit: The naming of this variable confused me. When I saw it, I thought it was storing the last error log, but the type was instant, which didn't make sense. Only when reading `record_get_err` did I realize it's being used to throttle/suppress/sample log messages. Maybe something like `err_log_sample_rate` or something? Some docs would be helpful, too.

Reply:
ack - I'll rename to `last_err_log_instant` and add a comment. `err_log_sample_rate` doesn't seem right either since it's the time we wrote the last err log rather than the rate at which we're writing them